onc/+util/extractFileExtension.m

```diff
@@ -0,0 +1,13 @@
+function ext = extractFileExtension(filename)
```

Comment:
Instead of this function you can use: https://www.mathworks.com/help/matlab/ref/fileparts.html which also handles string and char array.

Reply:
Oh this function is a perfect fit! I will update the codes